qubesadmin/tools/qvm_start_daemon.py

```diff
@@ -262,14 +365,32 @@ def common_guid_args(self, vm):
         if vm.features.check_with_template('rpc-clipboard', False):
             guid_cmd.extend(['-Q'])
 
+        guivm = self.app.domains[self.app.local_name]
```

Comment:
This should be rather `vm.guivm`. In practice it is the same value (gui-daemon is started only if vm.guivm is local), but it will be clearer this way.

Reply:
Done.